<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SF-LAP-497" failures="1" tests="5" name="L4L.TestCase.LoginPageTest" time="49.599" errors="1" timestamp="2023-08-17T19:58:28 IST" skipped="0">
  <testcase classname="L4L.TestCase.LoginPageTest" name="setup" time="7.329"/>
  <testcase classname="L4L.TestCase.LoginPageTest" name="validateLoginUrlTest" time="0.052"/>
  <testcase classname="L4L.TestCase.LoginPageTest" name="logintitletest" time="0.007"/>
  <testcase classname="L4L.TestCase.LoginPageTest" name="loginWithBlankCredentialsTest" time="0.150">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at L4L.TestCase.LoginPageTest.loginWithBlankCredentialsTest(LoginPageTest.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- loginWithBlankCredentialsTest -->
  <testcase classname="L4L.TestCase.LoginPageTest" name="ddlogintest" time="42.061">
    <error message="Unresolved compilation problems: 
	The declared package &quot;dd.l4l.pages&quot; does not match the expected package &quot;L4L.DD.pages&quot;
	The import dd cannot be resolved
	The import dd cannot be resolved
	L4LBaseclass cannot be resolved to a type
	DDUtil cannot be resolved
	driver cannot be resolved to a variable
	driver cannot be resolved
	DDUtil cannot be resolved
	driver cannot be resolved to a variable
	driver cannot be resolved
	DDUtil cannot be resolved
	driver cannot be resolved to a variable
	driver cannot be resolved
	DDUtil cannot be resolved
	driver cannot be resolved to a variable
	driver cannot be resolved
	DDUtil cannot be resolved
	driver cannot be resolved
	DDUtil cannot be resolved
	driver cannot be resolved to a variable
	driver cannot be resolved
	DDUtil cannot be resolved
	DDUtil cannot be resolved
	driver cannot be resolved to a variable
	driver cannot be resolved
	DDUtil cannot be resolved
	driver cannot be resolved to a variable
	driver cannot be resolved
	DDUtil cannot be resolved
	DDUtil cannot be resolved
	driver cannot be resolved to a variable
	DDUtil cannot be resolved
	driver cannot be resolved to a variable
	driver cannot be resolved
	DDUtil cannot be resolved
	DDUtil cannot be resolved
	driver cannot be resolved to a variable
" type="java.lang.Error">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
The declared package "dd.l4l.pages" does not match the expected package "L4L.DD.pages"
The import dd cannot be resolved
The import dd cannot be resolved
L4LBaseclass cannot be resolved to a type
DDUtil cannot be resolved
driver cannot be resolved to a variable
driver cannot be resolved
DDUtil cannot be resolved
driver cannot be resolved to a variable
driver cannot be resolved
DDUtil cannot be resolved
driver cannot be resolved to a variable
driver cannot be resolved
DDUtil cannot be resolved
driver cannot be resolved to a variable
driver cannot be resolved
DDUtil cannot be resolved
driver cannot be resolved
DDUtil cannot be resolved
driver cannot be resolved to a variable
driver cannot be resolved
DDUtil cannot be resolved
DDUtil cannot be resolved
driver cannot be resolved to a variable
driver cannot be resolved
DDUtil cannot be resolved
driver cannot be resolved to a variable
driver cannot be resolved
DDUtil cannot be resolved
DDUtil cannot be resolved
driver cannot be resolved to a variable
DDUtil cannot be resolved
driver cannot be resolved to a variable
driver cannot be resolved
DDUtil cannot be resolved
DDUtil cannot be resolved
driver cannot be resolved to a variable

at L4L.DD.pages.StudentTrackerPage.<init>(StudentTrackerPage.java:1)
at L4L.DD.pages.LoginPage.ddlogin(LoginPage.java:57)
at L4L.TestCase.LoginPageTest.ddlogintest(LoginPageTest.java:58)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- ddlogintest -->
</testsuite> <!-- L4L.TestCase.LoginPageTest -->
